package.json

```diff
@@ -49,7 +49,9 @@
   "resolutions": {
 	  "nth-check": "^2.0.1",
 	  "glob-parent": "^6.0.2",
-	  "trim": "^1.0.1"
+	  "trim": "^1.0.1",
+    "postcss": "^8.4.13",
+    "autoprefixer": "^10.4.7"
```

Comment:
@creador-dev there's no need to include `autoprefixer`

Reply:
@iamleigh Without autoprefixer postcss v8 throws an error.